### ISSUE TITLE
Showlist Page "Fillter Show Name" Doesn't Seem to be Functional 

### ISSUE THREAD

**Issue by DrakeJones** [SickRage/SickRage#3130]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

  SickRage Info:
Branch: master
Commit: 7e6e0575542c3f3e5476663532b1357f3ae1e902
Version: v2017.02.2-1
Database Version: 44.0

  Python Version:
2.7.10 (default, May 23 2015, 09:44:00) [MSC v.1500 64 bit (AMD64)]
  SSL Version:
OpenSSL 1.0.2a 19 Mar 2015
  OS:
Windows-7-6.1.7601-SP1
  Locale:
en_GB.cp1252
  User:
SERVER$
  Program Folder:
C:\SickRage\SickRage
  Config File:
C:\SickRage\Data\config.ini
  Database File:
C:\SickRage\Data\sickbeard.db
  Cache Folder:
C:\SickRage\Data\cache
  Log Folder:
C:\SickRage\Data\Logs
  Arguments:
['--nolaunch', '--port=8081', '--datadir=C:\\SickRage\\Data']

What you did:  Entered Show Name - Hit Enter Key
What happened: Nothing
What you expected: Filter Out non-selected names
Logs:
```
N/A
```

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by miigotu** [comment]
Make sure this is still an issue for you after the update.  Also, need to let me know what browser you are using

**Comment by DrakeJones** [comment]
Uh oh... Updated and got "{
"status":"false"
}"

Dead in the water.

Running outdated Firefox 46.0.1 because they keep fucking with the code and crippling  extensions and addons...

**Comment by miigotu** [comment]
Idk what you mean by status: false

**Comment by DrakeJones** [comment]
That text, verbatim, is what I see in my browser.  That’s it. No graphics – just that msg.  Last week I made a change to the settings on the “general” page and got that result. Had to use a backup file to recover the program.

**Comment by miigotu** [comment]
Try rebooting the pc

**Comment by DrakeJones** [comment]
If I try to open sr in ie I get this

**Comment by DrakeJones** [comment]
Okay now, Bud. Rebooted and SR is back up. SHOWS/Filter Show Name still non-functional…

**Comment by miigotu** [comment]
Idk, maybe your browser isn't allowing the js to run? Check firebug. Old browsers aren't really supported.

**Comment by sharkykh** [comment]
Hey @miigotu , he's actually not the only one.
I does work when you're in Poster view,
but not in any of the other ones. (Simple, Banner, Small Poster)

I look into it later today and see if I can come up with a fix.

**Comment by miigotu** [comment]
@sharkykh that text input is not supposed to be showing unless it is poster view, so only the column filter is supposed to be functional there

**Comment by miigotu** [comment]
On every layout other than poster you use the filter row under the column heading, for poster view there is no filter row thats why there is a filter input up top.
Fixed and pushed to master.

**Comment by DrakeJones** [comment]
Thanks Folks.

**Comment by sharkykh** [comment]
## __TL;DR:__ Not technical reasons, just more confusing.
Before I saw @miigotu 's comment I was attempting to implement so that it would filter the list, just like when you enter something in the text box under 'Show'.
I think it only makes it more confusing.
You either **discard** the current filter in order to filter by the value of 'Filter Show Name' (and no other column) or you add that value to the current applied filter.
On the former, if you have a saved filter that you like to use, it gets deleted and you have to write it again.
And on the latter if you have other columns filtered, like Status/Active/etc. and the show you are trying to filter doesn't match them, than you're not going to get the show that you want.
It's just adds more confusion instead of simplifying...
Kinda like a Lose-Lose situation, in my opinion.

**Comment by DrakeJones** [comment]
That seems a bit convoluted. But, if it’s that dangerous, maybe the search feature should just be removed altogether. ;-)

**Comment by sharkykh** [comment]
That's what miigotu changed it so it's only for poster view because it's the only one without a table and a filter.

**Comment by DrakeJones** [comment]
I see.
